sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -100,32 +108,162 @@ object Webpack {
     * @param log Logger
     * @return The generated bundles
     */
-  def bundle(
-    generatedWebpackConfigFile: File,
-    customWebpackConfigFile: Option[File],
-    webpackResources: Seq[File],
-    entries: Seq[(String, File)],
-    targetDir: File,
-    log: Logger
+  def libraries(
+      generatedWebpackConfigFile: File,
+      customWebpackConfigFile: Option[File],
+      webpackResources: Seq[File],
+      entryPointFiles: Seq[(String, File)],
+      entries: Seq[(String, File)],
+      targetDir: File,
+      log: Logger
   ): Seq[File] = {
 
     val configFile = customWebpackConfigFile
       .map(Webpack.copyCustomWebpackConfigFiles(targetDir, webpackResources))
       .getOrElse(generatedWebpackConfigFile)
 
-    log.info("Bundling the application with its NPM dependencies")
+    log.info("Bundling the applications NPM dependencies")
     Webpack.run("--config", configFile.absolutePath)(targetDir, log)
 
-    val bundles =
-      entries.map { case (key, _) =>
-        // TODO Support custom webpack config file (the output may be overridden by users)
-        targetDir / Webpack.bundleName(key)
-      }
-    bundles
+    entries.map {
+      case (k, v) =>
+        targetDir / Webpack.libraryFileName(k)
+    }
   }
 
+  val loaderScript: String =
+    """
+      |var exports = {};
```

Comment:
The point is *not* to have people use `globalFallback`. Otherwise how could they possibly write facades in any meaningful way, that is not *tight* to scalajs-bundler's quirks? That makes no sense.

`globalFallback` is to fall back on global variables for no-module. Not to serve what should be a CommonJS module but twisted into not being one for performance of the compilation.